src/window.rs

```diff
@@ -262,35 +249,14 @@ impl Window {
         }
     }
 
-    #[cfg(
-        any(
-            target_os = "linux",
-            target_os = "freebsd",
-            target_os = "dragonfly",
-            target_os = "openbsd"
-        )
-    )]
-    fn platform_builder_ext(window_builder: WindowBuilder, wm_class: &str) -> WindowBuilder {
+    #[cfg(not(any(target_os = "macos", windows)))]
```

Comment:
Shouldn't `windows` be in quotes as `"windows"` here?

Oh... it's not a `target_os` but a whole `cfg` of it's own. This seems weird, but probably has a reason? Not an issue with this PR.

Reply:
I'm not familiar with Rust's internals, but I believe this is related to operating system internals.

FreeBSD, OpenBSD, macOS, Linux are all at least  unix-ish, but Windows is a completely separate architecture. That's probably why `#[cfg(windows)]` and `#[cfg(unix)]` exist, but I don't think `#[cfg(macos)]` does.

Reply:
Interesting, thanks for some insight.